Core/Vault.php

```diff
@@ -189,4 +192,19 @@ private static function _getAddressId($oOrder)
     {
         return sha1(implode($oOrder->getShippingAccountHolder()->mappedProperties()));
     }
+
+    /**
+     * @param array $aCard
+     * @param array $aVaultCard
+     *
+     * @return bool
+     *
+     * @since 1.3.0
+     */
+    private static function _areCardsEqual($aCard, $aVaultCard)
+    {
+        return $aCard['TOKEN'] === $aVaultCard['token']
```

Comment:
why are here the array fields once in uppercase the other time in camel case?

Reply:
$aCard is created by the DatabaseProvider and here the keys are uppercase, $aVaultCard is created by us and there we use our coding standard